haystack/schema.py

```diff
@@ -94,17 +96,52 @@ def __str__(self):
         return str(self.to_dict())
 
 
+@dataclass
+class Answer:
+    answer: Optional[str]
+    type: Literal["generative", "extractive"]
+    #question: Optional[str]
+    score: Optional[float]
+    #probability: Optional[float] = None
+    context: Optional[str]
+    offset_start: Optional[int]
+    offset_end: Optional[int]
```

Comment:
in tables context can represent the whole table (if not too large). The offset will then represent the starting cell and end cells. As we might have multiple cells and in future multiple spans in a text let's go to:

offsets: [{"start": int, "end": int}, ...]